src/components/DiffView/index.tsx

```diff
@@ -197,17 +204,29 @@ export class DiffViewBase extends React.Component<Props> {
         messages = selectedMessageMap.byLine[line];
       }
 
-      let widget = null;
+      let widget = enableCommenting && line && !change.isDelete && (
+        <CommentList
+          addonId={version.addon.id}
+          fileName={version.selectedPath}
+          line={line}
+          versionId={version.id}
+        >
+          {(commentList) => <div>{commentList}</div>}
```

Comment:
This can be `{(allComments) => allComments}`. That is, there's no need for a div and in fact it's already [wrapped in one](https://github.com/mozilla/addons-code-manager/blob/83eb299a13464e5f25a1bbd47a4fae917eac3f6b/src/components/CommentList/index.tsx#L106). 

It may seem weird to need a render prop for this but if you consider wanting to render a container element around the comments but only when they exist, there's actually no other way to do that.